.licenseignore

```diff
@@ -0,0 +1,11 @@
+**/.swiftformat
+*.pb.swift
+*.txtpb
+.gitignore
+.gitmodules
+.swiftformat
+CITATION.cff
+Package.resolved
+Package.swift
```

Comment:
`swift-tools-version` is supposed to be the first line
`https://github.com/swiftlang/swift-evolution/blob/main/proposals/0152-package-manager-tools-version.md#how-the-swift-tools-version-is-specified`